src/CoreWCF.Primitives/src/CoreWCF/Channels/IAuthorizationCapabilities.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Channels;
+
+public interface IAuthorizationCapabilities
```

Comment:
Make this interface and its implementation internal and move to shared code folder